nucypher/network/server.py

```diff
@@ -113,56 +113,57 @@ def _make_rest_app(datastore: Datastore, this_node, domain: str, log: Logger) ->
 
     @rest_app.route("/public_information")
     def public_information():
-        """
-        REST endpoint for public keys and address.
-        """
-        response = Response(
-            response=bytes(this_node),
-            mimetype='application/octet-stream')
-
+        """REST endpoint for public keys and address."""
+        response = Response(response=bytes(this_node), mimetype='application/octet-stream')
         return response
 
-    @rest_app.route("/ping", methods=['POST'])
+    @rest_app.route("/ping", methods=['GET', 'POST'])
```

Comment:
May be worth splitting into two methods since they don't share any logic. I think Flask supports 
```
@rest_app.route("/ping", methods=['GET'])
def ping_get(): # or whatever
    ...
@rest_app.route("/ping", methods=['POST'])
def ping_post():
    ...
```

Reply:
While they don't share logic (although they can), they are very similar in purpose.